packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -265,7 +265,6 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, Acco
             index,
             contentRef,
             canBeCollapsed,
-            id: titleId,
```

Comment:
Oops, I think that is intentional. Can we revert this change?

Reply:
Some goes for line 280?